commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ReleaseNotes.java

```diff
@@ -164,6 +164,7 @@
     <li class=new-in-release>Added {@link io.sphere.sdk.orderedits.OrderEdit} update and staged actions</li>
     <li class=new-in-release>Added {@link io.sphere.sdk.orderedits.commands.OrderEditApplyCommand}</li>
     <li class=new-in-release>Added new {@link io.sphere.sdk.orderedits.OrderEdit} message types</li>
+    <li class=new-in-release><Added {@link ProductCatalogData#getCurrentUnsafe()} method/li>
```

Comment:
the closing `</li>` is missing not correct.

Reply:
Good catch !